tests/integration/test_xrdsst.py

```diff
@@ -146,6 +147,15 @@ def step_update_service_parameters(self, client_id):
             assert description["services"][0]["timeout"] == 120
             assert description["services"][0]["url"] == 'http://petstore.xxx'
 
+    def step_add_service_endpoints(self, client_id):
+        with XRDSSTTest() as app:
+            endpoint_controller = EndpointController()
+            endpoint_controller.app = app
+            endpoint_controller.load_config = (lambda: self.config)
+            endpoint_controller.add_endpoints()
+            description = get_service_description(self.config, client_id)
+            assert len(description["services"][0]["endpoints"]) == 1
```

Comment:
The same as in end to end tests, as this OPENAPI3 description also generated 4 endpoints, you should add the following assertions instead:

            assert len(description["services"][0]["endpoints"]) == 5
            assert str(description["services"][0]["endpoints"][4]["path"]) == "/testPath"
            assert str(description["services"][0]["endpoints"][4]["method"]) == "POST"